p5js/YOLO/YOLO_single_image/sketch.js

```diff
@@ -0,0 +1,53 @@
+// Initialize the Image Classifier method with MobileNet. A callback needs to be passed.
+// Create a YOLO method
+const yolo = ml5.YOLO(modelReady);
+let img;
+let objects = [];
+let status;
+
+function setup() {
+
+  createCanvas(400, 266);
+  img = createImg('images/turtle.png', imageReady);
+  img.hide();
+  img.size(400, 266);
+}
+
+// Change the status when the model loads.
+function modelReady() {
+  console.log("model Ready!")
+  status = true;
+  document.querySelector('#status').innerHTML = 'Model Loaded'
```

Comment:
I would suggest if it's a p5 example using p5.dom's `select()` method? I'm also noticing that the inclusion of extra stuff in `index.html` is a confusion point for beginners and just having the `console.log` is probably good enough for the examples. People can design their own more thoughtful HTML interfaces if they want?